src/control/XournalMain.cpp

```diff
@@ -32,6 +32,9 @@
 #endif
 
 #include <algorithm>  // std::sort
+#include <filesystem>
+
+namespace fs = std::filesystem;
```

Comment:
Replace with 
```
#include "filesystem.h"
``` 
Remove the fs namespace alias.

Reply:
FYI the file you will include is https://github.com/xournalpp/xournalpp/blob/pr-freeze/filesystem/src/filesystem.h.in . But don't Rebase your code upon this branch, because this branch needs a Rebase first.

Reply:
Done.